File: linux/arithmetic/gcc/arithmetic.cc
#include <iostream>

int main()
{
	int v1 = 41;
	int v2 = 7;
	int sum = v1 + v2;
	int sub = v1 - v2;
	int mul = v1 * v2;
	int quo = v1 / v2;
	int rem = v1 % v2;
	
	std::cout << sum << " | " << sub << " | " << mul << " | " << quo << " | " << rem << "\n";
	return 0;
}

File: linux/arithmetic/rust/arithmetic.rs
fn main() {
	let mut v1: u32 = 0;
	v1 = 41;
	let mut v2: u32 = 0;
	v2 = 7;
	let sum: u32 = v1 + v2;
	let sub: u32 = v1 - v2;
	let mul: u32 = v1 * v2;
	let quo: u32 = v1 / v2;
	let rem: u32 = v1 % v2;

	println!("{sum} | {sub} | {mul} | {quo} | {rem}");
}
